### ISSUE TITLE
linear scaling lr rule

### ISSUE THREAD

**Issue by hust-kevin** [open-mmlab/mmdetection#351]
```
 Linear Scaling Rule: When the minibatch size is multiplied by k, multiply the learning rate by k 
```
@hellock 

the minibatch size mean batchsize of per gpu or total size of total gpu? for example, in your setting, (#per_img_gpu, #num_gpu)=(2,8) and lr=0.02,  I have 1 gpu, I set per_img_gpu=2 ,so I should keep lr=0.2 or change lr to 0.02/8?

**Comment by hellock** [comment]
The batch size means #gpus * #images_per_gpu = 16. You need to set lr as 0.02/8 if using only one gpu.

**Comment by hust-kevin** [comment]
If batch size =#gpu *#images_per_gpu, that’s mean bn compute statistics across GPUs? But normal bn can achieve this? or in mmdetection bn means sync bn

**Comment by hellock** [comment]
The lr scaling has nothing to do with bn. Sync bn is currently not supported in mmdetection.

**Comment by sowmya04101998** [comment]
hi im trying to run the code for different number of gpus ,i did read this section 

*Important*: The default learning rate in config files is for 8 GPUs and 2 img/gpu (batch size = 8*2 = 16). According to the [Linear Scaling Rule](https://arxiv.org/abs/1706.02677), you need to set the learning rate proportional to the batch size if you use different GPUs or images per GPU, e.g., lr=0.01 for 4 GPUs * 2 img/gpu and lr=0.08 for 16 GPUs * 4 img/gpu.    
However i want to run the training with 1,2,4,8,16,24,28,32,36,40 gpus with images per gpu set as 4 , please let me know the lr i should be setting in each case
